io.gitlab.librewolf-community.json

```diff
@@ -0,0 +1,84 @@
+                "mv librewolf.tar.bz2 /app/lib/librewolf",
+                "install -D -m644 /app/lib/librewolf/browser/chrome/icons/default/default128.png /app/share/icons/hicolor/128x128/apps/io.gitlab.librewolf-community.png",
+                "install -D -m644 /app/lib/librewolf/browser/chrome/icons/default/default128.png /app/usr/share/icons/hicolor/128x128/apps/io.gitlab.librewolf-community.png",
+                "install -D -m644 /app/lib/librewolf/io.gitlab.librewolf-community.appdata.xml -t /app/share/metainfo",
+                "install -D -m644 io.gitlab.librewolf-community.desktop -t /app/share/applications",
+                "install -D -m755 librewolf.sh /app/bin/librewolf"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.com/librewolf-community/browser/linux/-/jobs/1267046474/artifacts/raw/LibreWolf-88.0.1-2.aarch64.tar.bz2",
+                    "sha256": "660284b741f6d2507e5d1eb3d423ad96577f59b7902c3028d189742c8cdb7424",
+                    "dest": "librewolf.tar.bz2",
+                    "strip-components": 0,
+                    "only-arches": [
+                        "aarch64"
+                    ]
+                },
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.com/librewolf-community/browser/linux/-/jobs/1265492102/artifacts/raw/LibreWolf-88.0.1-2.x86_64.tar.bz2",
+                    "sha256": "d16fc62c8cbce90854de8793523d6da946dacc195edfc4abef6452fcce05651e",
+                    "dest": "librewolf.tar.bz2",
+                    "strip-components": 0,
+                    "only-arches": [
+                        "x86_64"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "path": "io.gitlab.librewolf-community.desktop",
+                    "dest-filename": "io.gitlab.librewolf-community.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "librewolf.sh",
+                    "dest-filename": "librewolf.sh"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pcsc",
+        "--socket=pulseaudio",
+        "--persist=.librewolf",
+        "--persist=.mozilla",
+        "--filesystem=xdg-download:rw",
+        "--filesystem=xdg-run/pipewire-0",
+        "--device=dri",
+        "--filesystem=~/.config/dconf:ro",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--talk-name=org.freedesktop.FileManager1",
+        "--system-talk-name=org.freedesktop.NetworkManager",
+        "--talk-name=org.a11y.Bus",
+        "--talk-name=org.gnome.SessionManager",
+        "--talk-name=org.freedesktop.ScreenSaver",
+        "--talk-name=org.gtk.vfs.*",
+        "--socket=session-bus",
```

Comment:
why you need the whole session-bus?

Reply:
That was part of an attempt to mitigate some font rendering issues (somewhat related: https://github.com/flatpak/flatpak/issues/2861). Didn't help, though – I'll remove it and see if everything else keeps working :)